migration/sql-files/023-refactor-identities-users.sql

```diff
@@ -0,0 +1,14 @@
+-- Remove default value of identity IDs. IDs should not be generated by DB. IDs from the keycloak tokens should be used instead
```

Comment:
Oops, this comment is not quite relevant anymore. We still plan to generate identities IDs for remote WIs. But for keyclaok ones we will use the original KC IDs.
Regarding the existing data, I thought we agreed in Brno that just nuking the DB will be the best solution atm, no? :) The existing data in our demo DB doesn't seem to be 100% compatible with our planner UI anyway.